ui/webui/test/README.rst

```diff
@@ -129,11 +126,59 @@ Make sure to set::
 
     GITHUB_BASE=rhinstaller/anaconda
 
-before running any commands suggested there. For updating pixel test reference images you can use
-the available the Makefile target::
+For updating (pushing) updated pixel test reference images you can use the available the Makefile target::
+
+    make update-reference-images
+
+How to fix failed pixel tests
+-----------------------------
+
+For all the steps below you have to be in `ui/webui` directory of the project.
+
+Locally just copy the broken tests images to the `test/refercence` directory. However, easier
+option to deal with this is to use automation which will download all the broken images from
+fail test on PR::
+
+    ./test/common/pixel-tests fetch <link to HTML with failed tests>
+
+Example of such a call::
+
+    ./test/common/pixel-tests fetch https://cockpit-logs.us-east-1.linodeobjects.com/pull-4551-20230322-101308-479c2fc1-fedora-rawhide-boot-rhinstaller-anaconda
+
+The link will be link accessible from the `Details` button on GitHub PR with failed tests.
+
+When the images are correctly updated just call to push the changes to pixel repository
+(no review is required)::
 
     make update-reference-images
 
+Then new commit is pushed to
+["anaconda pixel tests repository"](https://github.com/rhinstaller/pixel-test-reference)
+and just add reference git submodule to your existing PR by::
+
+    git add test/reference
+    git commit
+    git push <your fork>
+
+If everything went well your PR should be green now.
+
+Outdated Cockpit CI image for testing
+-------------------------------------
+
+From time to time you can face an issue that the fedora-X-boot image on Cockpit side is
+missig dependency for your PR. **You should not push your PR without fixing the image first!**
+
+To update the image please ping #cockpit on IRC and they will provide a PR with the new image.
+It will look similar to ["this"](https://github.com/cockpit-project/bots/pull/4551).
```

Comment:
Does this mean that cockpit team will build a special image with the required dependencies, based on the anaconda PR? Because just grabbing newer images won't work.

Reply:
It should work as discussed today. Our tests depends on having the dependencies in official repositories the same as Cockpit does. At least AFAIK.

Reply:
Repository != image.